SG.js

```diff
@@ -529,3 +561,8 @@ loadRegdateCache();
 function saveRegdateCache() {
 	fs.writeFileSync('config/regdate.json', JSON.stringify(regdateCache));
 }
+
+function getEXPType(pokemon) {
+	let data = JSON.parse(fs.readFileSync('config/SGGame-Data/EXP.json', 'utf8'));
+	return data[pokemon].expType;
```

Comment:
Yeah I edited that in a later commit actaully, now its killing travis though i got a possible fix once I finish up my WIP commit :v